coreutils/whoami.rs

```diff
@@ -1,19 +1,8 @@
+use crate::libbb::appletlib::bb_show_usage;
+use crate::libbb::bb_pwd::xuid2uname;
+use crate::libbb::xfuncs_printf::fflush_all;
+use libc::{c_char, c_int, geteuid};
```

Comment:
nit: I've found that `{...}` imports make it much more difficult to do global find/replace refactoring. It may be less pretty with one-per-line, but it will save time in the future!